### PR TITLE
Changing SOCKETS to WEBSOCKETS

### PR DESCRIPTION
**Pull request by rahulrumalla**
Per @JeeChoi

### CODE REVIEW

**Comment by captDaylight** [comment]
@rahulrumalla let me know when it's good to go

**Comment by rahulrumalla** [comment]
Will do. Just clarifying some things with Jee

**Comment by rahulrumalla** [comment]
@captDaylight - can you merge but not close, please? Want to check visuals. 
@JeeChoi - can you review this now?

File: docs/wss/eth_subscription.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# eth_subscription
```
Comment (JeeChoi) [inline docs/wss/eth_subscription.md]: ummm.... where did you get this api??
I'm not sure you tested the example code here (or how you got the responses??)
>wscat -c wss://mainnet.infura.io/ws 
>{"id": 1, "method": "eth_subscription", "params": ["0x1150eb72fbeb2114c96bafd474ad8623"]}
returns:
{"jsonrpc":"2.0","id":1,"error":{"code":-32601,"message":"The method eth_subscription does not exist/is not available"}}

Comment (rahulrumalla) [reply docs/wss/eth_subscription.md]: Good catch! Misunderstood the endpoint - didn't test the individual subs call. Didn't realise its dynamic until you pointed out. I fixed it now. Others are tested as well.

**Pull request merged**